pkg/sosreport/org.cockpit-project.cockpit-sosreport.metainfo.xml

```diff
@@ -2,7 +2,7 @@
 <component type="addon">
   <id>org.cockpit_project.cockpit_sosreport</id>
   <metadata_license>CC0-1.0</metadata_license>
-  <name>Diagnostic Reports</name>
+  <name>Diagnostic reports</name>
```

Comment:
If we adopt the navigation as title case PF4 recommendation, then this should remain Title Case.